Sources/Fullscreen/AdReporter/AdReporterConfirmationView.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  Copyright © 2018 FINN.no. All rights reserved.
+//
+
+import UIKit
+
+public class AdReporterConfirmationView: UIView {
+    
+    // MARK: - Private properties
+    
+    private lazy var titleLabel: UILabel = {
+        let label = Label(style: .title2)
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+    
+    private lazy var messageLabel: UILabel = {
+        let label = Label(style: .body(.licorice))
+        label.numberOfLines = 0
+        label.lineBreakMode = .byWordWrapping
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+    
+    private lazy var closeButton: UIButton = {
+        let button = Button(style: .flat)
+        button.translatesAutoresizingMaskIntoConstraints = false
+        return button
+    }()
+    
+    // MARK - Public properties
+    
+    public var title: String? {
+        get { return titleLabel.text }
+        set { titleLabel.text = newValue}
+    }
+    
+    public var message: String? {
+        get { return messageLabel.text }
+        set { messageLabel.text = newValue }
+    }
+    
+    public var buttonTitle: String? {
+        get { return closeButton.titleLabel?.text }
+        set { closeButton.setTitle(newValue, for: .normal) }
+    }
+    
+    // MARK: - Action
+    
+    public var action: (() -> Void)?
```

Comment:
Can we use a delegate pattern instead of a closure so it's consistent with the rest of FinniversKit?